lxd/instance/drivers/driver_lxc.go

```diff
@@ -8246,15 +8246,13 @@ func (d *lxc) Info() instance.Info {
 
 // Metrics returns the metric set for the LXC driver. It collects various metrics related to memory, CPU, disk, filesystem, and network usage.
 func (d *lxc) Metrics(hostInterfaces []net.Interface) (*metrics.MetricSet, error) {
-	state := instance.PowerStateStopped
```

Comment:
Shouldn't this be a (partial) git revert of a prior commit?

Reply:
yes it should. fixed.